### DISCUSSION TITLE
`G` is not present in docs.

### DISCUSSION THREAD

**Discussion by kajuburfi** [helix-editor/helix#1479] [Q&A]
This is a discussion for simple one-line questions, which don't need a separate discussion.

I hope this will clear things up in the discussions tab.

**Comment by kajuburfi** [comment]
Is `G` deliberately left out in the docs(For goto line number)?
I find `g` to be there, but not `G` in Normal mode for goto_line_number command.

**Comment by archseer** [comment]
I'd rather have individual discussions, that way it's easier to find an existing answer

**Reply by kajuburfi** [reply]
Alright, let me change the name.

**Comment by TornaxO7** [comment]
@python128 it looks like that it's now include in the [docs](https://docs.helix-editor.com/keymap.html). (could you please mark your question as answered? :))
